text2.html

```diff
@@ -32,7 +32,7 @@
     <link rel="manifest" href="/assets/images/favicon/manifest.json" />
     <meta name="msapplication-TileColor" content="#004af2" />
     <meta name="theme-color" content="#004af2" />
-    <title>Terms &amp;Conditions</title>
+    <title>Terms &amp; Conditions</title>
```

Comment:
can you please delete this space
```suggestion
    <title>Terms &amp; Conditions</title>
```